app/calendar_grid.py

```diff
@@ -0,0 +1,51 @@
+import calendar
```

Comment:
This file should be on app/routing/dates.py, see https://fastapi.tiangolo.com/tutorial/bigger-applications/